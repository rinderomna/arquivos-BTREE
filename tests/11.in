7 tipo2 binario11.bin indice11.bin 1
181 1988 13 "AM" NULO "FIAT" NULO
