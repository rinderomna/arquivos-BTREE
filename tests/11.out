MARCA DO VEICULO: HONDA
MODELO DO VEICULO: NXR160 BROS ESDD
ANO DE FABRICACAO: 2020
NOME DA CIDADE: MESQUITA
QUANTIDADE DE VEICULOS: 21

